app/services/key_rotator/attribute_encryption.rb

```diff
@@ -20,20 +20,13 @@ def uak
     end
 
     def encrypted_attributes
-      encrypted_email_attributes.merge!(other_encrypted_attributes)
-    end
-
-    def encrypted_email_attributes
-      email = EncryptedAttribute.new_from_decrypted(user.email, uak)
-      { encrypted_email: email.encrypted, attribute_cost: new_cost }
-    end
-
-    def other_encrypted_attributes
-      User::ENCRYPTED_ATTRIBUTES.each_with_object({}) do |attribute, result|
-        plain_attribute = user.public_send(attribute)
+      User.attribute_names.grep(/^encrypted_/).each_with_object({}) do |attribute, result|
```

Comment:
I would do this to get the array of encrypted attribute names:
```ruby
User.attribute_names.select { |attr| attr.start_with?('encrypted') }
```
It's 2x faster than `grep`.